CMSgov/mint-app
# [EASI-2426] Operational Solution Subtasks

# EASI-2426

## Changes and Description

- Implementation for Operational Solution Subtasks
  - GQL
  - Migration SQL
  - Store SQL
  - Store Methods
  - Resolver Methods
  - Solution Relation Access Control Implementation



## How to test this change

- Run operational solution subtask unit tests and observe that they all pass
- Use the operational solution subtask endpoints in Postman to:
  - Create an operational solution subtask(s)
  - Update the operational solution subtask(s)
  - Get the associated operational solution and confirm that it contains the created subtask(s)
  - Delete the created operational solution subtask(s)
  - Confirm that all endpoints behave as expected

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
